contracts/StakeManager.sol

```diff
@@ -127,31 +165,33 @@ contract StakeManager is Ownable {
      * @dev Reverts when `_time` is bigger than `MAX_LOCKUP_PERIOD`
      * @dev Reverts when `_time + block.timestamp` is smaller than current lock time.
      */
-    function lock(uint256 _time) external onlyVault noMigration {
+    function lock(uint256 _time) external onlyVault noMigration processEpoch {
         if (_time > MAX_LOCKUP_PERIOD) {
             revert StakeManager__InvalidLockupPeriod();
         }
         Account storage account = accounts[msg.sender];
         processAccount(account, currentEpoch);
-        if (block.timestamp + _time < account.lockUntil) {
-            revert StakeManager__DecreasingLockTime();
-        }
```

Comment:
I might be missing something but this looks like one can now reduce the lockup time fro what it was set previously.
If that's the case, then this is a bug (unit tests should catch this), because this shouldn't be allowed.

Reply:
Before -> lock(uint256 absoluteTimeOfLock); Now -> lock(uint256 increasedTimeToLock)

Reply:
But indeed that might happen, will fix